apps/backend/src/modules/reviews/review.repository.ts

```diff
@@ -15,23 +21,64 @@ class ReviewRepository implements Repository {
 
 		const review = await this.reviewModel
 			.query()
-			.insert({
-				content,
-				likesCount,
-				poiId,
-				routeId,
-				userId,
-			})
+			.insert({ content, likesCount, poiId, routeId, userId })
 			.returning("*")
 			.execute();
 
 		return ReviewEntity.initialize(review);
 	}
 
-	public async findAll(): Promise<ReviewEntity[]> {
-		const reviews = await this.reviewModel.query().execute();
+	public async findAll(
+		options: null | ReviewSearchQuery,
+	): Promise<ReviewGetByIdResponseDto[]> {
+		const queryBuilder = this.reviewModel
+			.query()
+			.withGraphJoined("[user(selectUser).avatar(selectAvatar)]")
+			.modifiers({
+				selectAvatar(builder) {
+					builder.clearSelect().select("files.url as url");
+				},
+				selectUser(builder) {
+					builder
+						.clearSelect()
+						.select(
+							"users.id",
+							"users.first_name as firstName",
+							"users.last_name as lastName",
+						);
+				},
+			})
+			.select(
+				"reviews.id as id",
+				"reviews.content as content",
+				"reviews.likes_count as likesCount",
+				"reviews.poi_id as poiId",
+				"reviews.route_id as routeId",
+			)
+			.orderBy("reviews.id", "desc");
```

Comment:
We already have a enum for this, and it's better to order by created_at (from newest to oldest)
```suggestion
			.orderBy("reviews.created_at.", SortOrder.DESC);
```